app/api/helpers/pentabarfxml.py

```diff
@@ -0,0 +1,82 @@
+from flask import url_for
+from pentabarf.Conference import Conference
+from pentabarf.Day import Day
+from pentabarf.Event import Event
+from pentabarf.Person import Person
+from pentabarf.Room import Room
+from sqlalchemy import DATE
+from sqlalchemy import asc
+from sqlalchemy import cast
+from sqlalchemy import func
+
+from app import db
+from app.models.session import Session
+from app.settings import get_settings
+from app.models.event import Event as EventModel
+from app.models.microlocation import Microlocation
+
+
+class PentabarfExporter:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def export(event_id):
+        """Takes an event id and returns the event in pentabarf XML format"""
+        event = EventModel.query.get(event_id)
```

Comment:
What if the event doesn't exist i.e. a wrong event_id is sent ?

Reply:
@srv-twry Thanks for pointing that out. Using safe_query in the celery task method now. Please review :)